cli/tools/standalone.rs

```diff
@@ -209,6 +223,18 @@ async fn create_standalone_binary(
   Ok(final_bin)
 }
 
+fn set_windows_binary_to_gui(mut bin: Vec<u8>) -> Vec<u8> {
+  let header_start = LittleEndian::read_i32(&bin[60..64]);
```

Comment:
Thanks, I didn't know `align_to` was a thing.
I removed the crate and used your suggestion.
Also wrapped it into two functions, `read_as` and `read_as_mut`.
As I need to read multiple times and want to prevent size mismatches.